pkg/crds/calico/crd.projectcalico.org_felixconfigurations.yaml

```diff
@@ -306,7 +306,6 @@ spec:
                   are auto-detected.
                 type: string
               floatingIPs:
-                default: Disabled
```

Comment:
I'm not entirely sure where this change was picked up from, it doesn't seem at all related to the wireguard changes

Reply:
That was from master it looks like: https://github.com/tigera/operator/commit/82459ad7f232e5bc1b195e06564d0e3c82ae3ade#diff-b6824b8776988360d61c4a8e3c38552c52037ef281cac3cfdff0ed4882e70a27L308-L311

Reply:
Got it, thanks @lmm!